src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/notification/ClaimIssueNotificationHandlerTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.unspec.handler.callback.notification;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CallbackType;
+import uk.gov.hmcts.reform.unspec.config.properties.notification.EmailTemplates;
+import uk.gov.hmcts.reform.unspec.config.properties.notification.NotificationsProperties;
+import uk.gov.hmcts.reform.unspec.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.unspec.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.unspec.model.BusinessProcess;
+import uk.gov.hmcts.reform.unspec.sampledata.PartyBuilder;
+import uk.gov.hmcts.reform.unspec.service.NotificationService;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.unspec.handler.callback.notification.ClaimIssueNotificationHandler.NOTIFY_DEFENDANT_SOLICITOR_FOR_CLAIM_ISSUE_TASK_ID;
+
+@SpringBootTest(classes = {
+    ClaimIssueNotificationHandler.class,
+    CaseDetailsConverter.class,
+    JacksonAutoConfiguration.class
+})
+class ClaimIssueNotificationHandlerTest extends BaseCallbackHandlerTest {
+
+    @MockBean
+    private NotificationService notificationService;
+    @MockBean
+    private NotificationsProperties notificationsProperties;
+    private EmailTemplates emailTemplates;
+
+    @Autowired
+    private ClaimIssueNotificationHandler handler;
+
+    @Nested
+    class AboutToSubmitCallback {
+        @BeforeEach
+        void setup() {
+            emailTemplates = new EmailTemplates();
+            when(notificationsProperties.getEmailTemplates()).thenReturn(emailTemplates);
+        }
+
+        @Test
+        void shouldNotifyDefendantSolicitor_whenInvoked() {
+            Map<String, Object> data = new HashMap<>();
+            data.put("businessProcess", BusinessProcess.builder()
```

Comment:
I'd use `Map.of()` here, just for readability and maybe then put it in `HashMap` if it's necessary.

Reply:
done